packages/react-ai/src/components/AIConversation/__tests__/createAIConversation.test.tsx

```diff
@@ -1,15 +1,111 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { createAIConversation } from '../createAIConversation';
-import { avatars, messages } from '../mocks/mocks';
+import { ConversationMessage, SendMessage } from '../../../types';
+import { Avatars } from '../types';
+import { convertBufferToBase64 } from '../utils';
+
+const messages: ConversationMessage[] = [
+  {
+    conversationId: 'foobar',
+    id: '1',
+    content: [{ text: 'I am your virtual assistant' }],
+    role: 'assistant',
+    createdAt: new Date(2023, 4, 21, 15, 23).toDateString(),
+  },
+  {
+    conversationId: 'foobar',
+    id: '2',
+    content: [
+      {
+        text: 'I have a really long question. This is a long message This is a long message This is a long message This is a long message This is a long message',
+      },
+    ],
+    role: 'user',
+    createdAt: new Date(2023, 4, 21, 15, 24).toDateString(),
+  },
+  {
+    conversationId: 'foobar',
+    id: '3',
+    content: [
+      { image: { format: 'png', source: { bytes: new Uint8Array([]) } } },
+    ],
+    role: 'assistant',
+    createdAt: new Date(2023, 4, 21, 15, 25).toDateString(),
+  },
+];
+
+const avatars: Avatars = {
+  user: {
+    username: 'User',
+    avatar: (
+      <svg viewBox="0 0 24 24">
+        <path d="M10,10.9c2.373,0,4.303-1.932,4.303-4.306c0-2.372-1.93-4.302-4.303-4.302S5.696,4.223,5.696,6.594C5.696,8.969,7.627,10.9,10,10.9z M10,3.331c1.801,0,3.266,1.463,3.266,3.263c0,1.802-1.465,3.267-3.266,3.267c-1.8,0-3.265-1.465-3.265-3.267C6.735,4.794,8.2,3.331,10,3.331z"></path>
+        <path d="M10,12.503c-4.418,0-7.878,2.058-7.878,4.685c0,0.288,0.231,0.52,0.52,0.52c0.287,0,0.519-0.231,0.519-0.52c0-1.976,3.132-3.646,6.84-3.646c3.707,0,6.838,1.671,6.838,3.646c0,0.288,0.234,0.52,0.521,0.52s0.52-0.231,0.52-0.52C17.879,14.561,14.418,12.503,10,12.503z"></path>
+      </svg>
+    ),
+  },
+  ai: {
+    username: 'Raven',
+    avatar: (
+      <img
+        src={convertBufferToBase64(
```

Comment:
non-blocking, but this byte array and converting it to base64 is not really important for testing/integration because its just going into an img tag.